### ISSUE TITLE
Style (or otherwise indicate) unread tabs in v5?

### ISSUE THREAD

**Issue by fncll** [mbnuqw/sidebery#740]
My apologies if I am overlooking this option, but I am looking for a way to style (or otherwise indicate) unread tabs in the tree. I tried using some code found in an earlier issue here, but it might only work for earlier versions of the plugin, as it has no effect when I use it in the Sidebar Styles editor:

```.Tab[data-unread="true"] { background-color: #FFFF00; }```

Thanks!

**Comment by jathek** [comment]
That works for me in v5.
![image](https://user-images.githubusercontent.com/692914/187510820-8a69cf3f-8e4f-42fa-aa5d-91f7eb035b5c.png)

**Comment by guigirl42** [comment]
> works for me in v5

Oh, golly ... unfortunately, does _not_ work for me in 5.0.0b25. 🤷‍♀️
***

@fncll 
Fwiw, this is how i've styled it in TST:

![](https://i.postimg.cc/C5yXBYb9/2022-09-16-12-53-28.png)


via:

```
/* Show title of unread tabs with pink italic font
tab-item.unread .label-content {
  color: #ff93fe !important;
  font-style: italic;
  font-size: 14px;
}


/* Add private browsing indicator per tab */
.tab.private-browsing .label:before {
  content: "🕶";
}
/* Show Private Browsing tabs with distinctive green background */
.tab.private-browsing .label {
  background-color: #00ff7f;
}


/* Highlight active tab. This makes the active tab very noticeable increasing its height and modifying the color and font
tab-item.active tab-item-substance {
  height: 26px !important;
}  
tab-item.active .background {  
  background-color: #aaaaff;
}
tab-item.active .label-content {
  font-weight: bold;
  font-size: 16px;
}
tab-item.active tab-twisty,
tab-item.active .label-content,
tab-item.active tab-counter {
  color: #5907FF;  
}


/* Change styling of pending (unloaded) tabs 
tab-item.discarded tab-item-substance {
  opacity: 0.75;
  font-style: italic;
  font-size: 14px;  
}
tab-item.discarded .label-content {
  /* color: #818162; */
  color: #3DEDD8;
}


/* Control font-size of background tabs */
tab-label {
  font-size: 14px;  
}
```

, but disappointingly all my attempts so far to find a way to mimic this in Sidebery, have failed.

**Comment by jathek** [comment]
> Oh, golly ... unfortunately, does _not_ work for me in 5.0.0b25. 🤷‍♀️
> , but disappointingly all my attempts so far to find a way to mimic this in Sidebery, have failed.

![image](https://user-images.githubusercontent.com/692914/190632261-d8cdb798-68a1-4ba7-8c7a-556b24c957db.png)

I based the above on your provided code. Some of your code I didn't do because I didn't want to search for the Sidebery analogues, but you get the idea - it all works. I would double-check if the code you have in Sidebery has errors - when I copied the code from your comment into VS Code, it showed syntax errors from improperly commented lines.

![image](https://user-images.githubusercontent.com/692914/190632713-a5b98452-a77c-4d1a-8da3-980cbdeea489.png)

Working code from my screenshot (**for v5**):
```css
/* Show title of unread tabs with pink italic font */
.Tab[data-unread="true"] .title {
  color: #ff93fe;
  font-style: italic;
  font-size: 14px;
}
/* Highlight active tab. This makes the active tab very noticeable increasing its height and modifying the color and font */
.Tab[data-active="true"] {  
  background-color: #aaaaff;
}
.Tab[data-active="true"] .title {
  font-weight: bold;
  font-size: 16px;
}

/* Change styling of pending (unloaded) tabs */
.Tab[data-discarded="true"] .title {
  color: #3DEDD8;
}
```

**Comment by guigirl42** [comment]
@nearcatch -- **genius**, _thanks so much_! 

As you can see, i'm not much good at this stuff, but your corrected CSS has put a smile on my face.

**Comment by mbnuqw** [comment]
@nearcatch, thanks your help, I hope I can close this issue.

**Comment by GodKratos** [comment]
Is this still the correct syntax for v5.3.3?
I've added the code above to styles, the active and discarded tabs can be changed but I can't seem to affect a change to unread tabs at all.

**Comment by silverwings15** [comment]
yes it's correct. i'm currently using ```.Tab[data-unread="true"] .title {font-style: italic;}``` and it works just fine

**Comment by GodKratos** [comment]
Ah, what I've realised is that the css is working, however the "data-unread" status of a tab is only true when the tab is newly created. I'm used to tabs being "unread" when you haven't viewed them after a page has been refreshed or loaded another URL.

This issue seems to have been captured in https://github.com/mbnuqw/sidebery/issues/913 but the fix that was applied doesn't resolve the actual problem as page refreshes are not affected and I've tried using the "data-updated" field in css and don't seem to be getting any affect for new URLs either.
